multinode/roles/monitoring_install/tasks/main.yml

```
@@ -0,0 +1,133 @@
+- name: Copy the promtail config file /usr/local/bin for fullnode servers
+  become: true
+  template:
+    src: "{{ playbook_dir }}/templates/promtail-config.j2"
+    dest: "/usr/local/bin/promtail-config.yaml"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+  loop: "{{ range(0, fullnode_count) | list }}"
+  loop_control:
+    index_var: item
+  vars:
+    node_name:  "fullnode{{ item }}"
```

yamllint (1.29.0-1)

[warning] too many spaces after colon

(colons)